librz/debug/dsession.c

```diff
@@ -64,7 +64,9 @@ RZ_API bool rz_debug_add_checkpoint(RzDebug *dbg) {
 	for (i = 0; i < RZ_REG_TYPE_LAST; i++) {
 		RzRegArena *a = dbg->reg->regset[i].arena;
 		RzRegArena *b = rz_reg_arena_new (a->size);
-		memcpy (b->bytes, a->bytes, b->size);
+		if (a->size) {
```

Comment:
is this check really needed? I think rz_mem_copy just do nothing if a->size = 0, right?

Reply:
You're right but I don't want to print assertions from `rz_mem_copy`(and those should stay there since dest/src could be NULL with size >0 somewhere else in the future)
 
```
WARNING: rz_mem_copy: assertion 'dest && src' failed (line 46)                                                        
WARNING: rz_mem_copy: assertion 'dest && src' failed (line 46)                                                        
```